documentation/general/dash-sonic-hld.md

```diff
@@ -140,7 +139,7 @@ Metering is essential for billing the customers and below are the high-level req
 	- Mapping table based metering - E.g For specific destinations within the mapping table that must be billed separately     
 - Policy in the metering context refers to metering policy associated to Route tables. This is not related to ACL policy or any ACL packet counters.  
 - If packet flow hits multiple metering buckets, order of priority shall be **Policy->Route->Mapping**
-- User shall be able to override the precedence between Routing/Policy and Mapping buckets by setting an _override_ flag. When policy is enabled for a route, it takes higher precedence than routing and mapping metering bucket unless _override_ flag is set in which case Mapping takes precedence
+- User shall be able to override the precedence between Routing and Mapping buckets by setting an _override_ flag. When policy is enabled for a route, it takes higher precedence than routing and mapping metering bucket. 
```

Comment:
Are the changes to this section, acl tags and fastpath sections intentional ? It seems to be reverting a prior commit. Can you plz check ?